aptos/api/aux-ts/src/graphql/pyth.ts

```diff
@@ -27,6 +35,79 @@ export function getRecognizedTVL(coinType: string, amount: number): number {
   return 0;
 }
 
+export async function getUsdPrice(
```

Comment:
do you need pyth? i had to comment it out because the serum endpoint was erroring out with too many 522s and failing http requests and the solana endpoint no longer supports pyth

there was some discussion a while back, was there a solution?

Reply:
thanks!